src/styles/blocks/page.css

```diff
@@ -0,0 +1,68 @@
+.page {
+  font-family: "Manrope", sans-serif;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 22px;
+  color: #1b2129;
+
+  scroll-behavior: smooth;
```

Comment:
The 'scroll-behavior' property should be applied to the 'html' or 'body' tag, not the '.page' class. Move the 'scroll-behavior: smooth;' declaration to the '.page__body' class.